src/query/service/src/pipelines/executor/pipeline_executor.rs

```diff
@@ -178,11 +178,11 @@ impl PipelineExecutor {
             }
         }
 
-        if let Err(error_code) = self.graph.check_finished() {
-            let may_error = Some(error_code);
-            (self.on_finished_callback)(&may_error)?;
-            return Err(may_error.unwrap());
-        }
+        // if let Err(error_code) = self.graph.check_finished() {
```

Comment:
Remove this or leave comments here?

Reply:
I will uncomment it in other PR(we need it, but it will cause an error when canceling the task).